### PR TITLE
Fixing table being announced for button

### PR DESCRIPTION
**Pull request by aasimkhan30**
Issue:  #12961 

This PR adds a role presentation to the formComponent action div. This prevents the screen reader from reading actions as table.

### CODE REVIEW

**Comment by alanrenmsft** [comment]
@aasimkhan30 the issue you linked doesn't seem to be related to the PR, could you please fix it?

**Comment by aasimkhan30** [comment]
@alanrenmsft, fixed the linked issue

File: extensions/import/src/common/constants.ts [hunk 1]
```diff
@@ -32,8 +32,10 @@ export const serverDropDownTitleText = localize('flatFileImport.serverDropdownTi
 export const databaseDropdownTitleText = localize('flatFileImport.databaseDropdownTitle', "Database the table is created in");
 export const invalidFileLocationError = localize('flatFile.InvalidFileLocation', "Invalid file location. Please try a different input file");
 export const browseFilesText = localize('flatFileImport.browseFiles', "Browse");
+export const browseFileAriaLabel = localize('flatFileImport.browseFileAriaLabel', "Browse for a file to import");
 export const openFileText = localize('flatFileImport.openFile', "Open");
 export const fileTextboxTitleText = localize('flatFileImport.fileTextboxTitle', "Location of the file to be imported");
+export const fileTextboxArialabel = localize('flatFileImport.fileTextboxArailabel', "Location of the file to be imported, you can also use the browse button below to select a file");
```
Comment (alanrenmsft) [inline extensions/import/src/common/constants.ts:38]: aria label is supposed to be short, I think `File to import` is good enough.

**Comment by alanrenmsft** [comment]
it is not obvious to me how this will fix the linked issue, could you please add more description?

File: src/sql/workbench/browser/modelComponents/formContainer.component.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ class FormItem {
 							<model-component-wrapper [descriptor]="item.descriptor" [modelStore]="modelStore" [style.width]="getComponentWidth(item)" [style.height]="getRowHeight(item)">
 							</model-component-wrapper>
 						</div>
-						<div *ngIf="itemHasActions(item)" class="form-item-row form-actions-table form-item-last-row">
+						<div *ngIf="itemHasActions(item)" class="form-item-row form-actions-table form-item-last-row" role="presentation">
```
Comment (alanrenmsft) [inline src/sql/workbench/browser/modelComponents/formContainer.component.ts:77]: this should be enough, do you want to keep other changes?

**Comment by Charles-Gagnon** [comment]
As well as a better description of what is changing please also include a screenshot of the previous layout. 

Is this fixed with just adding the role? If not then we may want to do something more so other people don't run into this issue when making UI in the extension.

**Comment by aasimkhan30** [comment]
@Charles-Gagnon , updated the description.

**Review by alanrenmsft — APPROVED** [review]

**Review by Charles-Gagnon — APPROVED** [review]

**Review by kburtram — APPROVED** [review]

**Pull request merged**